docs/reference/dashboard-monitoring-with-elastic-agent.md

```diff
@@ -86,6 +86,7 @@ Check out [Installing {{agent}}](docs-content://reference/fleet/install-elastic-
 6. Configure the integration to collect metrics.
 
     * Make sure that **Metrics (Elastic Agent)** is turned on (default), and **Metrics (Stack Monitoring)** is turned off.
+   * Disabling certain metrics might result in incomplete dashboard visualizations. Therefore, we recommend collecting all metricsets.
```

Comment:
```suggestion
   * For the best experience with the Logstash dashboards, we recommend collecting all of the metrics. Turning off metrics will result in incomplete or missing visualizations.
```

Reply:
Sounds good!
I have replaced Logstash with `{{ls}}` to align with the standard.